src/parachain/fee.ts

```diff
@@ -0,0 +1,74 @@
+import { decodeFixedPointType } from "..";
+import { PolkaBTC, UnsignedFixedPoint } from "../interfaces";
+import { DefaultOracleAPI, OracleAPI } from "./oracle";
+import Big from "big.js";
+import { ApiPromise } from "@polkadot/api";
+
+/**
+ * @category PolkaBTC Bridge
+ */
+export interface FeeAPI {
+    /**
+     * @param amountSat Amount, in Satoshi, for which to compute the required
+     * griefing collateral, in Planck
+     * @param griefingCollateralRate
+     * @param oracleAPI
+     * @returns The griefing collateral, in Planck
+     */
+     getGriefingCollateralInPlanck(
+        amountSat: PolkaBTC,
+        griefingCollateralRate: UnsignedFixedPoint
+    ): Promise<Big>;
+    /**
+     * @param feesPolkaBTC Satoshi value representing the BTC fees accrued
+     * @param feesDOT Planck value representing the DOT fees accrued
+     * @param lockedDOT Planck value representing the value locked to gain yield
+     * @param dotToBtcRate Conversion rate
+     * @returns The APY, given the parameters
+     */
+    calculateAPY(feesPolkaBTC: string, feesDOT: string, lockedDOT: string, dotToBtcRate: Big): string;
+    /**
+     * @returns The griefing collateral rate for issuing PolkaBTC
+     */
+    getIssueGriefingCollateralRate(): Promise<UnsignedFixedPoint>;
```

Comment:
I think this should return a string or a `Big`. I think the conversion should happen inside the lib.